crates/runtime/src/instance.rs

```diff
@@ -47,6 +47,20 @@ cfg_if::cfg_if! {
                 self.instance().signal_handler.set(Some(Box::new(handler)));
             }
         }
+    } else if #[cfg(target_os = "fuchsia")] {
```

Comment:
FWIW this is fine to leave out, this is intended for the API extensions in the `wasmtime` crate but those were left out, so this could be a noop on fuchsia.